api/src/backend/api/migrations/0066_finding_resource_group_scanresourcegroupsummary.py

```diff
@@ -0,0 +1,127 @@
+import uuid
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+import api.db_utils
+import api.rls
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("api", "0065_alibabacloud_provider"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="finding",
+            name="resource_group",
+            field=models.CharField(
```

Comment:
I'd add a `TextField` just in case. I know 50 chars should be more than enough but I prefer using that one.

Reply:
Done